### PR TITLE
Remove kcli package dependencies

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

File: 00_prepare.sh [hunk 1]
```diff
@@ -18,5 +13,3 @@ gpgcheck=0""" > /etc/yum.repos.d/racadm.repo
 dnf -y install openssl-devel srvadmin-idracadm7
 
 systemctl enable --now httpd
-kcli create host kvm -H 127.0.0.1 local
-kcli create pool -p /var/www/html default
```
Comment (yuvalk) [inline 00_prepare.sh:22]: I'm creating this so I wont have to move the iso to the httpd directory

File: helpers/racadm.sh [hunk 2]
```diff
@@ -1,11 +0,0 @@
...
-SERVER=$2
-USER=$3
-PASSWORD=$4
-ISO_URL="http://$(hostname -I | cut -f1 -d" " | xargs)/$CLUSTER.iso"
-/opt/dell/srvadmin/bin/idracadm7 -r $SERVER -u $USER -p $PASSWORD remoteimage -s &&\
-/opt/dell/srvadmin/bin/idracadm7 -r $SERVER -u $USER -p $PASSWORD remoteimage -d &&\
-/opt/dell/srvadmin/bin/idracadm7 -r $SERVER -u $USER -p $PASSWORD remoteimage -c -l $ISO_URL &&\
-/opt/dell/srvadmin/bin/idracadm7 -r $SERVER -u $USER -p $PASSWORD set iDRAC.VirtualMedia.BootOnce 1 &&\
-/opt/dell/srvadmin/bin/idracadm7 -r $SERVER -u $USER -p $PASSWORD set iDRAC.ServerBoot.FirstBootDevice VCD-DVD &&\
-/opt/dell/srvadmin/bin/idracadm7 -r $SERVER -u $USER -p $PASSWORD serveraction powercycle
```
Comment (yuvalk) [inline helpers/racadm.sh]: y did you remove this file? where will it come from?

File: kcli_plan.yml [hunk 3]
```diff
@@ -23,10 +21,9 @@
   - 01_create_iso.sh
   - 02_install_baremetal.sh
   - sno_params.yml
-  - origin: helpers/racadm.sh
```
Comment (yuvalk) [inline kcli_plan.yml:26]: file is located in helpers/racadm.sh
if you wanted to move it, it is missing from this commit.

File: 00_prepare.sh [hunk 4]
```diff
@@ -18,5 +13,5 @@ gpgcheck=0""" > /etc/yum.repos.d/racadm.repo
...
+
+test -f /root/.ssh/id_rsa.pub || ssh-keygen -t rsa -N '' -f /root/.ssh/id_rsa
```
Comment (yuvalk) [inline 00_prepare.sh:17]: for?

File: kcli_plan.yml [hunk 5]
```diff
@@ -23,10 +22,14 @@
...
+  - origin: racadm.sh
     path: /root/bin/racadm.sh
   - origin: {{ pull_secret }}
     path: /root/openshift_pull.json
   scripts:
   - deploy.sh
-
+{% if installer_wait %}
+  finishfiles:
```
Comment (yuvalk) [inline kcli_plan.yml]: what's finishfiles?
anyhow, can plz be a separate commit?

**Review by yuvalk — CHANGES_REQUESTED** [review]

File: 00_prepare.sh (hunk 4 above)
Comment (karmab) [reply 00_prepare.sh:17]: this allows to deploy even when privtekey is set to False (which should be the default value since otherwise we re exposing the private key of the hypervisor in the vm, with no real benefit)

File: kcli_plan.yml (hunk 5 above)
Comment (karmab) [reply kcli_plan.yml]: moved to https://github.com/yuvalk/kcli-sno/pull/2

File: 00_prepare.sh (hunk 4 above)
Comment (yuvalk) [reply 00_prepare.sh:17]: :+1:
